tasks/main.yml

```diff
@@ -1,5 +1,35 @@
 ---
 
+
+- name: Show Clean Insights measurement collection disclaimer
+  debug:
+    msg: |
+      By continuing with the ongoing installation process,
+      you are giving your consent for the collection of the following
+      Clean Insights measurements:
+        - Installed OpenWISP Version
+        - Enabled OpenWISP modules: A list of the enabled OpenWISP modules
+          along with their respective versions
+        - OS details: Information on the operating system, including its version, kernel version, and platform
+
+      If you prefer not to participate, you can opt-out by configuring
+      "openwisp2_clean_insights_measurement_consent: false" in your playbook.
+      However, before making that decision, we encourage you to explore how this
+      information contributes to the enhancement of OpenWISP at the following link:
+      https://github.com/openwisp/openwisp-utils#clean-insights-measurements
```

Comment:
The problem we have now is that the README is not being rendered.

We'll need to add a page in the openwisp2-docs repo for this, publish it and link that page.